scenario/migrate_vms.yaml

```diff
@@ -0,0 +1,85 @@
+namespace:
```

Comment:
Please add comments or docstring describing the purpose of this scenario and when to use it